sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -8,10 +9,8 @@
 :toc: left
 :encoding: utf-8
 :lang: en
-:sectnums:
```

Comment:
There might not be precedent in the other docs with the new format we're referencing, but I think section numbers adds value in as large extension specification as ours, and have a slight preference to keeping them if possible.